README.md

```diff
@@ -191,20 +145,9 @@ However, WKWebViews do not display an address bar or an HTTPS lock icon. If the
 
 - PopupBridge attaches to a WKWebView by [injecting a user script](https://developer.apple.com/reference/webkit/wkusercontentcontroller/1537448-adduserscript) to the page
   - This exposes a JavaScript interface (via `window.popupBridge`) for the web page to interact with the iOS code
-- The web page detects whether the page has access to `window.popupBridge`; if so, it uses `popupBridge.open` to open the popup URL
-  - `popupBridge.open` creates a SFSafariViewController to open the popup URL and has its delegate present the view controller
+- The web page detects whether the page has access to `window.popupBridge`; if so, it creates a ASWebAuthenticationSession to open the popup URL
   - The web page can also use `popupBridge.onComplete` as a callback
-- The popup web page uses a deep link URL to dismiss the popup
```

Comment:
Did we check to see if a cancellation that comes from within the pop-up (such as the tiny cancelation text in the PP flow) properly cancels?

Reply:
We should check to see if any changes need to be made to the JS portion of the PopopBridge integration

Reply:
> Did we check to see if a cancellation that comes from within the pop-up (such as the tiny cancelation text in the PP flow) properly cancels?

Yep, tested that in cancels/dismisses the window and hits the same block that it did in 1.x branch.